x/collection/keeper/keeper_test.go

```diff
@@ -39,22 +43,30 @@ type KeeperTestSuite struct {
 	numRoots int
 }
 
-func createRandomAccounts(accNum int) []sdk.AccAddress {
-	seenAddresses := make(map[string]bool, accNum)
-	addresses := make([]sdk.AccAddress, accNum)
-	for i := 0; i < accNum; i++ {
-		var address sdk.AccAddress
-		for {
-			pk := secp256k1.GenPrivKey().PubKey()
-			address = sdk.AccAddress(pk.Address())
-			if !seenAddresses[address.String()] {
-				seenAddresses[address.String()] = true
-				break
+func (s *KeeperTestSuite) createRandomAccounts(accNum int) []sdk.AccAddress {
+	if s.deterministic {
+		addresses := make([]sdk.AccAddress, accNum)
+		for i := range addresses {
+			addresses[i] = sdk.AccAddress(fmt.Sprintf("address%d", i))
+		}
+		return addresses
+	} else {
+		seenAddresses := make(map[string]bool, accNum)
```

Comment:
It's just FYI. You may already know.
Map with an empty struct as a value like this `make(map[string]struct{}, accNum)` doesn't consume any memory.
And We can still check if a key exists in a conventional way `if _, ok := m["key"]; !ok {}`.

Reply:
As you know, it's one of the eternal debates on Go. If the size of the elements is sufficiently large, we should use `struct{}` as you suggested. But in this case, it's just a matter of choice. Thank you for reminding me.

Reply:
I agree. That works well in case of large one~